### ISSUE TITLE
🆕 Software Discussion | Clam AntiVirus

### ISSUE THREAD

**Issue by qssdsa** [privacytoolsIO/privacytools.io#1450]
### Clam AntiVirus
**Name:** [ClamAV](https://www.wikipedia.org/wiki/Clam_AntiVirus)
**Category:** Antivirus and General security
**URL:** https://www.clamav.net/
**Developer**: Varies, currently by [Cisco Systems](https://en.wikipedia.org/wiki/Cisco_Systems)

### ClamTk
**Name:** [ClamTk](https://www.wikipedia.org/wiki/ClamTk)
**Category:** Antivirus and General security
**URL:** https://gitlab.com/dave_m/clamtk/-/wiki_pages/Home
**Maintainer**: Dave Mauroni

**ClamWin**:
**Name:** [ClamWin](https://www.wikipedia.org/wiki/ClamWin_Free_Antivirus)

## Description

> Clam AntiVirus (ClamAV) is a free software, cross-platform and open-source antivirus software toolkit able to detect many types of malicious software, including viruses. One of its main uses is on mail servers as a server-side email virus scanner. - [Wikipedia](https://www.wikipedia.org/wiki/Clam_AntiVirus)

#### But anti-viruses are spyware!

Some could very well be, but ClamAV is FLOSS and searches viruses based on signatures.
**More Info**: https://www.wikipedia.org/wiki/Clam_AntiVirus#Unofficial_databases

**Comment by dawidpotocki** [comment]
> ### Clam AntiVirus
> **Name:** [ClamAV](https://www.wikipedia.org/wiki/Clam_AntiVirus)
> **Category:** Antivirus and General security
> **URL:** https://www.clamav.net/
> **Developer**: Varies, currently by [Cisco Systems](https://en.wikipedia.org/wiki/Cisco_Systems)
>
> ### ClamTk
> **Name:** [ClamTk](https://www.wikipedia.org/wiki/ClamTk)
> **Category:** Antivirus and General security
> **URL:** https://gitlab.com/dave_m/clamtk/-/wiki_pages/Home
> **Maintainer**: Dave Mauroni

I don't know how I feel about adding them, but I feel like they are more
for scanning Windows viruses, which doesn't seem that useful for OSes we
recommend.

> **ClamWin**:
> **Name:** [ClamWin](https://www.wikipedia.org/wiki/ClamWin_Free_Antivirus)

I don't think we should list Windows-only software as we don't recommend
Windows anyway.

**Comment by qssdsa** [comment]
> I don't know how I feel about adding them, but I feel like they are more for scanning Windows viruses, which doesn't seem that useful for OSes we recommend.
> **ClamWin**: **Name:** [ClamWin](https://www.wikipedia.org/wiki/ClamWin_Free_Antivirus)
> I don't think we should list Windows-only software as we don't recommend Windows anyway.

ClamAV appears to be mainly developed for UNIX-like OSes.
Windows versions of ClamAV appear to be ports made by 3rd parties.[[1](https://web.archive.org/web/20080720205113/http://www.clamav.net/download/packages/)][[2](https://archive.is/2019.11.01-053609/https://www.clamav.net/download)]

In fact, the bulk usage of ClamAV appears to be servers. 
I personally, attribute this to the popularity and simplicity of [Windows Defender](https://www.wikipedia.org/wiki/Windows_Defender), [VirusTotal](https://www.wikipedia.org/wiki/VirusTotal) or [Malwarebytes](https://www.wikipedia.org/wiki/Malwarebytes_(software)) have.

ClamAV has largely been a CLI only project until users started using projects like [ClamTk](https://www.wikipedia.org/wiki/ClamTk).
</br> https://www.wikipedia.org/wiki/Clam_AntiVirus#Graphical_interfaces

**Comment by dawidpotocki** [comment]
> ClamAV appears to be mainly developed for UNIX-like OSes.

Yes, but that doesn't mean that it doesn't look mostly for Windows
viruses.
